packages/browser/src/browser/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { AnalyticsSnippet } from './standalone-interface'
```

Comment:
Since file has import side effects -- I think we should name it 'global-analytics-helpers.ts ' or something like that, so nobody puts anything else in it. It could also go in /lib

Reply:
Good point. It stopped being only an `utils` file